Orange/widgets/gui.py

```diff
@@ -3240,3 +3241,16 @@ def setRange(self, minValue, maxValue, step=1.0):
         # For compatibility with qwtSlider
         # TODO If it's related to Qwt, remove it
         self.setScale(minValue, maxValue, step)
+
+
+class ControlGetter:
+    def __init__(self, widget):
+        self.widget = widget
+
+    def __getattr__(self, name):
+        widget = self.widget
+        if hasattr(widget, "settingsHandler") and \
```

Comment:
I guess that the hasattr part of the check is there to handle OWComponents, and the second part is there to handle multilevel gets?

If so, could you please add a comment stating that, as it not clear from the code. (at first I commented that the hasattr could be removed as widget always has a settingsHandler, but that is not the case for owcomponents)